File: src/main/java/com/goudourasv/institutions/controller/dto/InstitutionCreate.java
package com.goudourasv.institutions.controller.dto;

import javax.validation.constraints.NotBlank;

public class InstitutionCreate {
    @NotBlank
    private String name;
    @NotBlank
    private String country;
    @NotBlank
    private String city;

    public InstitutionCreate(String name,String country,String city) {
        this.name = name;
        this.country =country;
        this.city = city;
    }

    public String getName() {
        return name;
    }

    public String getCountry() {
        return country;
    }

    public String getCity() {
        return city;
    }
}

File: src/main/java/com/goudourasv/http/institutions/dto/InstitutionCreate.kt
package com.goudourasv.http.institutions.dto

import java.util.*
import javax.validation.constraints.NotBlank

data class InstitutionCreate(
    @NotBlank val name: String,
    @NotBlank val country: String,
    @NotBlank val city: String,
    val instructorIds: List<UUID> = emptyList()
)